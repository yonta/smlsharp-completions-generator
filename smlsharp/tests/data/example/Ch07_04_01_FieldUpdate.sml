fun f modify x = modify # {X = x};

fun reStructure (p as {Salary,...}) = p # {Salary=Salary * (1.0 - 0.0803)};

