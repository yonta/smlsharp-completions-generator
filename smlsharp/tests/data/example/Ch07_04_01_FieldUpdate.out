val f = fn : ['a#{X: 'b}, 'b. 'a -> 'b -> 'a]

val reStructure = fn : ['a#{Salary: real}. 'a -> 'a]
